db/migrate/20230317143105_add_omni_auth_columns_to_users.rb

```diff
@@ -0,0 +1,19 @@
+class AddOmniAuthColumnsToUsers < ActiveRecord::Migration[7.0]
+  def change
+    change_table :users, bulk: true do |t|
+
+      # Omniauthable, custom devise
+      t.string :auth_provider, null: false, default: ""
+      t.string :auth_subject_uid
+
+      ## Trackable
+      t.integer  :sign_in_count, default: 0, null: false
+      t.datetime :current_sign_in_at
+      t.datetime :last_sign_in_at
+      t.string   :current_sign_in_ip
+      t.string   :last_sign_in_ip
```

Comment:
Trackable: confirm if this (IP with other data) presents a GDPR issue for our use case.

Reply:
🤷   but this is part of the trackable module from devise so if we are wanting to keep an eye on who is using it from a security point of view it would seem to be needed. Can you look into the GDPR issue?

Reply:
Is there a GDPR issue with internal staff?